source/optional_features/delayed_capture/index.html.md.erb

```diff
@@ -37,3 +37,32 @@ payment flow. Otherwise, it will expire.
 If your service has enabled payment receipt emails, GOV.UK Pay will send a
 payment receipt email to the user once your service's capture request is
 received and accepted.
+
+## See the capture URL for a payment
+
+You can see the capture URL for a single payment in GOV.UK Pay API responses.
```

Comment:
The usefulness of the capture URL appearing is that it indicates both when a payment is available to capture and how to capture it, so it might be good to, er, capture that former angle in the lede with something like: “If a payment is available for capture, its capture URL will be included in the GOV.UK Pay API responses.”